packages/client/src/components/profile-link/profile-link.module.pcss

```
@@ -0,0 +1,47 @@
+.profileLink {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-right: 14px;
+}
+
+.profileLink.small {
+  padding-right: 0;
+}
```

В проекте есть [postcss-nested](https://github.com/postcss/postcss-nested)
```suggestion
.profileLink {
  display: flex;
  align-items: center;
  justify-content: space-between;
  padding-right: 14px;
  
  &.small {
    padding-right: 0;
  }
}
```